hack/trivy_scan.py

```diff
@@ -0,0 +1,318 @@
+# The script:
+# 1. Extract all the images used by the Kubeflow Working Groups
+# - The reported image lists are saved in respective files under ../docs/image_lists directory
```

Comment:
lets just create image-lists in the root and not rely on /docs, since it will be gone soon.

Reply:
> I created the image_lists folder in root and made the necessary changes in Trivy_scan.py file